business_requirement_deliverable_cost/models/business.py

```diff
@@ -57,10 +57,27 @@ def _get_partner(self):
             return False
 
     @api.multi
+    def _get_project(self):
+        self.ensure_one()
+        br_id = br_deliverable = False
+        if self.business_requirement_deliverable_id.id:
+            br_deliverable = self.business_requirement_deliverable_id
+        if br_deliverable.business_requirement_id.id:
+            br_id = br_deliverable.business_requirement_id
+        if br_id and br_id.project_id:
+            return br_id.project_id
+        else:
+            return False
+
+    @api.multi
     def _get_pricelist(self):
         self.ensure_one()
+        project_id = self._get_project()
         partner_id = self._get_partner()
-        if partner_id:
+        if project_id:
```

Comment:
Why not using `if project_id and project_id.pricelist_id:` (also applies for the `if` below)?